### PR TITLE
[AD-627] Add new method to get aggregate operations as strings

### PR DESCRIPTION
**Pull request by andiemontoyeah**
### Summary

- Added public method in DocumentDbMqlQueryContext to return aggregate operations as strings instead of Bson Documents

### Description

- Added unit test 
- Added public method in DocumentDbMqlQueryContext 

### Related Issue

https://bitquill.atlassian.net/browse/AD-627

### Additional Reviewers
@affonsoBQ
@andiem-bq
@alexey-temnikov
@birschick-bq

### CODE REVIEW

File: calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContext.java [hunk 1]
```diff
@@ -38,4 +40,18 @@
...
+     * Gets the aggregation operations (stages) for the query as a list of strings.
+     *
+     * @return the aggregation operations as an ordered list of strings in extended JSON format.
+     */
+    public List<String> getAggregateOperationsAsStrings() {
+        return aggregateOperations.stream()
+                .map(
+                        doc ->
+                                doc.toBsonDocument()
+                                        .toJson(JsonWriterSettings.builder().outputMode(JsonMode.EXTENDED).build()))
```
Comment (andiemontoyeah) [inline calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContext.java]: Using extended JSON format instead of toString() or default toJson() methods since these use standard JSON format. Extended JSON preserves BSON types.

**Review by affonsov — APPROVED** [review]
lgtm

File: calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContext.java (hunk 1 above)
Comment (birschick-bq) [reply calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContext.java]: nit: Do we need everything on its own line?

Comment (andiemontoyeah) [reply calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContext.java]: Nope. Just got carried away. 😅  Rearranged!

File: calcite-adapter/src/test/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContextTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                        + "\"includeArrayIndex\": \"array_index_lvl_0\", "
+                        + "\"preserveNullAndEmptyArrays\": true}}");
+        // Stage with different Bson types
+        stages.add(
+                "{\"$project\": {"
+                        + "\"literalTimestamp\": {\"$date\": {\"$numberLong\": \"1505938660000\"}}, "
+                        + "\"literalInt\": {\"$literal\": {\"$numberInt\": \"-2147483648\"}}, "
+                        + "\"literalDecimal\": {\"$literal\": {\"$numberDouble\": \"123.45\"}}, "
+                        + "\"literalVarchar\": {\"$literal\": \"Hello\"}, "
+                        + "\"literalBinary\": {\"$binary\": {\"base64\": \"RfCr\", \"subType\": \"00\"}}}}");
```
Comment (alinaliBQ) [inline calcite-adapter/src/test/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContextTest.java]: I wonder if it would be a good idea to add nulls in the checks? Just to get the corner cases?

Comment (andiemontoyeah) [reply calcite-adapter/src/test/java/software/amazon/documentdb/jdbc/query/DocumentDbMqlQueryContextTest.java]: Added some examples

**Review by alinaliBQ — APPROVED** [review]
lgtm!

**Review by birschick-bq — APPROVED** [review]
